Fix compilation error due to missing libgnomeui CFLAGS into DEPS_CFLAGS,
and undefined GNOME_CLIENT_PARAM_SM_CONNECT macroconstant due to missing
"libgnomeui/gnome-client.h" header file.

 configure.ac |    1 +
 src/main.cpp |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/configure.ac b/configure.ac
index 6a1280b..7db94ea 100644
--- a/configure.ac
+++ b/configure.ac
@@ -19,6 +19,7 @@ dnl checks for libraries
 PKG_CHECK_MODULES(DEPS, gconfmm-2.6 >= 2.6.0 \
                         gtkmm-2.4 >= 2.6.0 \
                         libgnomecanvasmm-2.6 >= 2.6.0 \
+			libgnomeui-2.0 >= 2.0.0 \
 			libglademm-2.4 >= 2.4.0 \
 			libpanelapplet-2.0 >= 2.0.0 \
                         libgtop-2.0 >= 2.6.0)
diff --git a/src/main.cpp b/src/main.cpp
index 73cc528..f7f8a00 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -28,6 +28,7 @@
 #include <gtkmm/main.h>
 
 #include <libgnomeui/gnome-ui-init.h>
+#include <libgnomeui/gnome-client.h>
 #include <panel-applet.h>
 
 #include "applet.hpp"
